src/app/chat/views/message_list.nim

```diff
@@ -103,7 +105,8 @@ QtObject:
       ChatMessageRoles.OutgoingStatus.int: "outgoingStatus",
       ChatMessageRoles.ResponseTo.int: "responseTo",
       ChatMessageRoles.Index.int: "index",
-      ChatMessageRoles.ImageUrls.int: "imageUrls"
+      ChatMessageRoles.ImageUrls.int: "imageUrls",
+      ChatMessageRoles.Image.int: "image"
```

Comment:
What's the difference between `imageUrls` and `image`?

Reply:
After looking below, it's when we receive only an image with no text?

Reply:
`imageUrls` is used with the feature to automatically display images when you include a link in the message
`image` is an attribute part of the Message object, that contains the base64 image when the content type of the message is of type image (`7`)